tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -64,3 +65,23 @@ Verify GPU Operator Deployment  # robocop: disable
     # ...   nvidia-driver-daemonset-49.84.202201212103-0 DS
     # ...   nvidia-node-status-exporter DS
     # ...   nvidia-operator-validator DS
+
+Verify Pytorch And Tensorflow Can Be Spawned
+    [Documentation]    Check Cuda builds are complete and  Verify Pytorch and Tensorflow can be spawned
+    [Tags]    Sanity
+    ...       ODS-480
+    Verify Cuda Builds Are Completed
+    Verify Image Can Be Spawned  image=pytorch  size=Default
+    Verify Image Can Be Spawned  image=tensorflow  size=Default
+
+
+*** Keywords ***
+Verify Cuda Builds Are Completed
+    [Documentation]    Verify All Cuda Builds have status as Complete
+    ${Pods} =    Run    oc get build -n redhat-ods-applications
+    @{builds} =    Split String    ${Pods}    \n
```

Comment:
Does this return only the cuda builds?

Reply:
Yes.

Reply:
Mh, I don't know if it will ever happen but there could be more than just the cuda builds at some point. Maybe you should have some filter on the name and check that it includes `cuda` or something similar?

Reply:
Changed